susePublicCloudInfoClient/pint

```diff
@@ -56,8 +56,8 @@ import sys
 
 from docopt import docopt
 
-import infoserverrequests
-import version
+import lib.susepubliccloudinfoclient.infoserverrequests as ifsrequest
+import lib.susepubliccloudinfoclient.version as version
 
```

Comment:
What would you suggest here that will work both for development & production use?